build/crd/condition.yaml

```diff
@@ -14,3 +14,11 @@
     Localhost - a profile defined in a file on the node should be used.
     RuntimeDefault - the container runtime default profile should be used.
     Unconfined - no profile should be applied.
+- op: add
+  path: "/spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/proxy/properties/pgBouncer/properties\
+    /containers/items/properties/securityContext/properties/seccompProfile/properties/type/description"
```

Comment:
⛏️ This operation and the one above it seem a bit out of place in a file named `condition.yaml`.

Reply:
I don't think the other files we use for similar purpose are great fits either. The only other options that come to mind are to either rename this file or add a new one, but I'm not sure I'm thrilled with those either. 
I think for this commit, I'd like to leave these here, but perhaps we can discuss a refactor later.